src/styles/theme-manager.js

```diff
@@ -191,8 +191,35 @@ module.exports = {
         backgroundColor: 'transparent',
         borderColor: rawTheme.palette.borderColor,
       },
+      isRtl: rawTheme.isRtl,
```

Comment:
But we have `styleConstants` used in some places without checking for the `isRtl` variable. This mainly includes animations I believe. Honestly we wouldn't need these here anyway if they weren't used in any place other that the `ensureDirection` function.